cmd/corectl/main.go

```diff
@@ -237,6 +237,10 @@ func createToken(db *sql.DB, args []string) {
 		fatalln("error:", err)
 	}
 	fmt.Println(tok.Token)
+
+	if *flagNet {
+		fmt.Println("warning: the network flag is deprecated")
```

Comment:
This message should go to stderr. Prob `fmt.Fprintln(os.Stderr, ...)` is the way to go.